src/browser/fixture.js

```diff
@@ -29,22 +29,29 @@ function getComposedChildren(node) {
 
 }
 
-async function waitForElem(elem) {
+async function waitForElem(elem, opts) {
 
 	const update = elem?.updateComplete;
 	if (typeof update === 'object' && Promise.resolve(update) === update) {
 		await update;
 		await nextFrame();
 	}
 
+	if (opts.awaitLoadingComplete && typeof elem?.getLoadingComplete === 'function') {
+		await elem.getLoadingComplete();
+		await nextFrame();
+	}
+
 	const children = getComposedChildren(elem);
-	await Promise.all(children.map(e => waitForElem(e)));
+	await Promise.all(children.map(e => waitForElem(e, opts)));
 
 }
 
 export async function fixture(element, opts) {
+	opts ??= {};
```

Comment:
Unless you foresee `null` being passed
```suggestion
export async function fixture(element, opts = {}) {
```